src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/NotificationService.java

```diff
@@ -206,35 +207,47 @@ public void sendContestedGeneralApplicationOutcomeEmail(CallbackRequest callback
             recipientEmail = (String) courtDetails.get(COURT_DETAILS_EMAIL_KEY);
         }
 
-        notificationRequest = createNotificationRequest(callbackRequest);
-        notificationRequest.setNotificationEmail(recipientEmail);
+        applicantNotificationRequest = createNotificationRequestForAppSolicitor(callbackRequest);
+        applicantNotificationRequest.setNotificationEmail(recipientEmail);
 
         URI uri = buildUri(notificationServiceConfiguration.getContestedGeneralApplicationOutcome());
-        sendNotificationEmail(notificationRequest, uri);
+        sendNotificationEmail(applicantNotificationRequest, uri);
     }
 
     private void sendNotificationEmail(NotificationRequest notificationRequest, URI uri) {
         HttpEntity<NotificationRequest> request = new HttpEntity<>(notificationRequest, buildHeaders());
         try {
             restTemplate.exchange(uri, HttpMethod.POST, request, String.class);
         } catch (Exception ex) {
-            log.error(MESSAGE,
-                notificationRequest.getCaseReferenceNumber(), MSG_SOLICITOR_EMAIL,
+            log.error(String.format("Failed to send email for case ID: %s for solicitor email: %s due to exception: %s",
+                notificationRequest.getCaseReferenceNumber(),
                 notificationRequest.getNotificationEmail(),
-                EXCEPTION, ex.getMessage());
+                ex.getMessage()));
         }
     }
 
-    private NotificationRequest createNotificationRequest(CallbackRequest callbackRequest) {
+    private NotificationRequest createNotificationRequestForAppSolicitor(CallbackRequest callbackRequest) {
         if (isConsentedApplication(callbackRequest.getCaseDetails())) {
-            notificationRequest = buildNotificationRequest(callbackRequest, SOLICITOR_REFERENCE,
+            applicantNotificationRequest = buildNotificationRequest(callbackRequest, SOLICITOR_REFERENCE,
                 CONSENTED_SOLICITOR_NAME, SOLICITOR_EMAIL, CONSENTED, GENERAL_EMAIL_BODY, DIVORCE_CASE_NUMBER);
         } else {
-            notificationRequest = buildNotificationRequest(callbackRequest, SOLICITOR_REFERENCE,
+            applicantNotificationRequest = buildNotificationRequest(callbackRequest, SOLICITOR_REFERENCE,
                 CONTESTED_SOLICITOR_NAME, CONTESTED_SOLICITOR_EMAIL, CONTESTED, GENERAL_EMAIL_BODY, DIVORCE_CASE_NUMBER);
         }
 
-        return notificationRequest;
+        return applicantNotificationRequest;
+    }
+
+    private NotificationRequest createNotificationRequestForRespSolicitor(CallbackRequest callbackRequest) {
+        if (isConsentedApplication(callbackRequest.getCaseDetails())) {
```

Comment:
I prefer ternary operators: 

applicantNotificationRequest = isConsentedApplication(callbackRequest.getCaseDetails()) ?
buildNotificationRequest(callbackRequest, RESP_SOLICITOR_REFERENCE, RESP_SOLICITOR_NAME, RESP_SOLICITOR_EMAIL, CONSENTED, GENERAL_EMAIL_BODY, DIVORCE_CASE_NUMBER) : buildNotificationRequest(callbackRequest, RESP_SOLICITOR_REFERENCE, RESP_SOLICITOR_NAME, RESP_SOLICITOR_EMAIL, CONTESTED, GENERAL_EMAIL_BODY, DIVORCE_CASE_NUMBER);

Reply:
Totally agree - changes made to use ternary operators 👍